app/components/views/GetStartedPage/ProcessUnmanagedTickets/ProcessUnmanagedTickets.jsx

```diff
@@ -8,14 +8,16 @@ import styles from "./ProcessUnmanagedTickets.module.css";
 import { useEffect } from "react";
 import { VSPSelect } from "inputs";
 
-export default ({ cancel, onSendContinue, onProcessTickets, title, description, noVspSelection }) => {
+export default ({ cancel, send, onProcessTickets, title, description, noVspSelection, isProcessingUnmanaged, error }) => {
   const [isValid, setIsValid] = useState(false);
   const [vsp, setVSP] = useState(null);
   const onSubmitContinue = async (passphrase) => {
-    // send a continue to go to the loading state
-    onSendContinue();
-    await onProcessTickets(passphrase, vsp.host, vsp.pubkey);
-    onSendContinue();
+    await onProcessTickets(passphrase, vsp.host, vsp.pubkey)
```

Comment:
i think function can normal func instead of async and then `return onPro...` - the then and catch should be
enough to resolve the promise.